src/screens/GroupSetting/elements/EditGroup.js

```diff
@@ -90,7 +90,7 @@ const styles = StyleSheet.create({
     borderRadius: normalize(48 / 2)
   },
   editName: {
-    color: COLORS.white,
+    color: COLORS.almostBlack,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the text in the `editName` style has been changed from `COLORS.white` to `COLORS.almostBlack`. This is a significant change and could affect readability depending on the background color. Make sure this change doesn't negatively impact the readability of the text.
```diff
- color: COLORS.white,
+ color: COLORS.almostBlack,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->